### PR TITLE
API Pod - Azure Development account

### PR DESCRIPTION
**Pull request by pataruco**
# API Pod - Azure Development account

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/rec6wEC66M7ccbNmi?blocks=hide)

### Acceptance Criteria

- [x] I want to connect to K8's pods using a public address

### Testing information

Please run this request on the terminal and you should receive a **200 OK**

```sh
curl -vvv  51.11.35.34/api/healthz
```
### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: infrastructure/README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ The following instructions are divided in:
 
 - [Provisioning infrastructure](./docs/infrastructure.md)
 - [Connect to Medicines microservices](./docs/kubernetes.md)
+- [Destroy and provision a Kubernetes cluster](./docs/destroy-provision-aks)
```
Comment (rjkerrison) [inline infrastructure/README.md]: needs to be `.md` in the link

File: infrastructure/environments/development/variables.tf [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+variable "CLIENT_ID" {
+  description = "Service Principal Client ID"
+}
+
+variable "CLIENT_SECRET" {
+  description = "Service Principal Client Secret"
+}
+
+variable "ENVIROMENT" {
```
Comment (rjkerrison) [inline infrastructure/environments/development/variables.tf]: typos `environment`

File: infrastructure/environments/non-prod/main.tf [hunk 3]
```diff
@@ -11,132 +11,40 @@ terraform {
...
-}
-
-resource "azurerm_storage_container" "cpd_website" {
-  name                  = "$web"
-  storage_account_name  = azurerm_storage_account.cpd.name
-  container_access_type = "container"
+locals {
+  resource_group_name = var.RESOURCE_GROUP_PRODUCTS
+  location            = var.REGION
+  environment         = var.ENVIROMENT
```
Comment (rjkerrison) [inline infrastructure/environments/non-prod/main.tf]: this `var.ENVIRONMENT` typo again

File: infrastructure/environments/prod/main.tf [hunk 4]
```diff
@@ -9,120 +9,36 @@ terraform {
...
-
-# waiting for this to be resolved: https://github.com/terraform-providers/terraform-provider-azurerm/issues/1903
-# (which is imminent), but in the meantime ...
-module "products_staticweb" {
-  source               = "git@github.com:StefanSchoof/terraform-azurerm-static-website.git"
-  storage_account_name = azurerm_storage_account.products.name
+locals {
+  resource_group_name = var.RESOURCE_GROUP_PRODUCTS
+  location            = var.REGION
+  environment         = var.ENVIROMENT
```
Comment (rjkerrison) [inline infrastructure/environments/prod/main.tf]: another `ENVIRONMENT` typo

File: infrastructure/modules/cluster/outputs.tf [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+output "public_ip" {
+  value = azurerm_public_ip.products_ip.ip_address
+}
+
+output "host" {
+  value = azurerm_kubernetes_cluster.cluster.kube_config.0.host
+}
+
+output resource_group_name {
```
Comment (rjkerrison) [inline infrastructure/modules/cluster/outputs.tf]: quotes for consistency with lines 1,5,9,13

File: infrastructure/modules/cluster/variables.tf [hunk 6]
```diff
@@ -0,0 +1,24 @@
+variable "resource_group_name" {
+  description = "Resource group name"
+}
+variable "location" {
+  description = "Resource group location"
+}
+variable "environment" {
+  description = "Enviroment name to use as a tag"
```
Comment (rjkerrison) [inline infrastructure/modules/cluster/variables.tf]: `Environment`

File: infrastructure/modules/cpd/variables.tf [hunk 7]
```diff
@@ -0,0 +1,13 @@
+variable "resource_group_name" {
+  description = "Resource group name"
+}
+variable "location" {
+  description = "Resource group location"
+}
+variable "environment" {
+  description = "Enviroment name to use as a tag"
```
Comment (rjkerrison) [inline infrastructure/modules/cpd/variables.tf]: `Environment`

File: infrastructure/modules/products/variables.tf [hunk 8]
```diff
@@ -0,0 +1,12 @@
+variable "resource_group_name" {
+  description = "Resource group name"
+}
+variable "location" {
+  description = "Resource group location"
+}
+variable "environment" {
+  description = "Enviroment name to use as a tag"
```
Comment (rjkerrison) [inline infrastructure/modules/products/variables.tf]: typo

File: infrastructure/README.md (hunk 1 above)
Comment (pataruco) [reply infrastructure/README.md]: Thanks, done!

File: infrastructure/environments/development/variables.tf (hunk 2 above)
Comment (pataruco) [reply infrastructure/environments/development/variables.tf]: Thanks, done

File: infrastructure/environments/non-prod/main.tf (hunk 3 above)
Comment (pataruco) [reply infrastructure/environments/non-prod/main.tf]: Thanks, done

File: infrastructure/environments/prod/main.tf (hunk 4 above)
Comment (pataruco) [reply infrastructure/environments/prod/main.tf]: Thanks, done

File: infrastructure/modules/cluster/outputs.tf (hunk 5 above)
Comment (pataruco) [reply infrastructure/modules/cluster/outputs.tf]: Thanks, done

File: infrastructure/modules/cluster/variables.tf (hunk 6 above)
Comment (pataruco) [reply infrastructure/modules/cluster/variables.tf]: Thanks

File: infrastructure/modules/cpd/variables.tf (hunk 7 above)
Comment (pataruco) [reply infrastructure/modules/cpd/variables.tf]: Thanks, done

File: infrastructure/modules/products/variables.tf (hunk 8 above)
Comment (pataruco) [reply infrastructure/modules/products/variables.tf]: thanks, done

**Pull request merged**